osu.Game.Tournament/JsonPointConverter.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using Newtonsoft.Json;
+
+namespace osu.Game.Tournament
+{
+    /// <summary>
+    /// We made a change from using SixLabors.ImageSharp.Point to System.Drawing.Point at some stage.
+    /// This handles converting to a standardised format on json serialize/deserialize operations.
+    /// </summary>
+    internal class JsonPointConverter : JsonConverter<Point>
+    {
+        public override void WriteJson(JsonWriter writer, Point value, JsonSerializer serializer)
+        {
+            // use the format of LaborSharp's Point since it is nicer.
+            serializer.Serialize(writer, new SixLabors.ImageSharp.Point(value.X, value.Y));
+        }
+
+        public override Point ReadJson(JsonReader reader, Type objectType, Point existingValue, bool hasExistingValue, JsonSerializer serializer)
+        {
+            if (reader.TokenType != JsonToken.StartObject)
```

Comment:
Oh, and `JsonToken.String` is also a token type, so we can make extra sure nobody shoves an integer in here or something.

Reply:
and do what on failure? the way this is done it will fail with a "value not string" error or something, whereas if it's opposite further checks will be required to throw something sane.

Reply:
It'll die on a cast, yeah. Just figured it doesn't hurt much to check but fine to leave as is I suppose.